code/model/modelrender.cpp

```diff
@@ -2628,12 +2629,16 @@ void model_render_queue(model_render_params* interp, model_draw_list* scene, int
 			pmi = model_get_instance(shipp->model_instance_num);
 		}
 		else if (pm->flags & PM_FLAG_HAS_INTRINSIC_ROTATE) {
-			if (objp->type == OBJ_ASTEROID)
+			if (objp->type == OBJ_ASTEROID) {
 				pmi = model_get_instance(Asteroids[objp->instance].model_instance_num);
-			else if (objp->type == OBJ_WEAPON)
+			} else if (objp->type == OBJ_WEAPON) {
 				pmi = model_get_instance(Weapons[objp->instance].model_instance_num);
-			else
+			} else if (objp->type == OBJ_JUMP_NODE) {
+				CJumpNode *jnp = jumpnode_get_by_objnum(objnum);
+				pmi = model_get_instance(jnp->GetPolymodelInstanceNum());
```

Comment:
An assert to make sure `jnp` is initialized here would be good:
```suggestion
				Assertion(jnp != nullptr, "Could not find jump node with object number %d!", objnum);
				pmi = model_get_instance(jnp->GetPolymodelInstanceNum());
```

Also, the `*` of a pointer type should be aligned to its type and not the variable.